8Knot/pages/index/index_callbacks.py

```
@@ -860,3 +860,37 @@ def hide_loading_on_landing(pathname):
 
 # Note: Landing page callbacks moved to pages/landing/landing_callbacks.py
+
+
+# Callback to change pill color when search is clicked
+@callback(
+    Output("projects", "className"),
+    [Input("search", "n_clicks"), Input("projects", "value")],
+    [State("projects", "className")],
+    prevent_initial_call=True,
+)
+def update_pill_color_on_search(search_clicks, selected_values, current_classname):
```

Ruff (0.14.3)

Unused function argument: `selected_values`

(ARG001)